packages/recommend-core/src/getFrequentlyBoughtTogether.ts

```diff
@@ -31,9 +32,14 @@ export function getFrequentlyBoughtTogether<TObject>({
   return recommendClient
     .getFrequentlyBoughtTogether<TObject>(queries)
     .then((response) =>
-      mapToRecommendations({
+      // Multiple identical recommended `objectID`s can be returned b
```

Comment:
Can we fix this typo while we're at this? (same applies below)

```suggestion
      // Multiple identical recommended `objectID`s can be returned by
```